docs/en/datasets/det.md

```diff
@@ -212,3 +218,26 @@ rm dataset.zip && rm -rf dataset
 ```bash
 python tools/data/textdet/funsd_converter.py PATH/TO/funsd --nproc 4
 ```
+
+### ILST
+- Step1: Download dataset from [here](https://iiitaphyd-my.sharepoint.com/:f:/g/personal/minesh_mathew_research_iiit_ac_in/EtLvCozBgaBIoqglF4M-lHABMgNcCDW9rJYKKWpeSQEElQ?e=zToXZP)
+- Step2: Run the following commands
+```bash
+unzip -q IIIT-ILST.zip && rm IIIT-ILST.zip
+cd IIIT-ILST
+# rename files
+cd Devanagari && for i in `ls`; do mv -f $i `echo "devanagari_"$i`; done && cd ..
+cd Malayalam && for i in `ls`; do mv -f $i `echo "malayalam_"$i`; done && cd ..
+cd Telugu && for i in `ls`; do mv -f $i `echo "telugu_"$i`; done && cd ..
+# transfer image path
+mkdir imgs && mkdir annotations
+mv Malayalam/{*jpg,*jpeg} imgs/ && mv Malayalam/*xml annotations/
+mv Devanagari/*jpg imgs/ && mv Devanagari/*xml annotations/
+mv Telugu/*jpeg imgs/ && mv Telugu/*xml annotations/
+# remove unnecessary files
+rm -rf Devanagari && rm -rf Malayalam && rm -rf Telugu && rm -rf README.txt
+```
+- Step3: Generate `instances_training.json` and `instances_val.json`(optional). Since the original dataset doesn't have a validation set, you may specify `--val-ratio` to split the dataset. E.g., if val-ratio is 0.2, then 20% of the data are left out as the validation set in this example.
```

Comment:
add a space blank between `instance_val.json` and (optional)

Reply:
ok